docs/using.rst

```diff
@@ -42,126 +22,35 @@ above method instead.
    https://github.com/letsencrypt/letsencrypt/archive/master.zip
 
 
-.. _prerequisites:
```

Comment:
Where is it referenced?  I can only find a reference in the letsencrypt-auto script :)